src/styles/main.css

```diff
@@ -249,7 +250,22 @@
 .form-switch-checkbox:checked + .form-switch-label:before {
   @apply right-0 bg-white;
 
-  margin-right: 0.3rem;
+  margin-right: 0.1rem;
+}
+
+#date, 
+#quantity {
+  width: 20px;
+  height: 20px;
+  border: 1px solid #e7e7e7;
+  border-radius: 2px;
+}
+
+#date:checked, 
+#quantity:checked {
+  background: white;
+  border: 6px solid #8acc9e;
+  border-radius: 2px;
```

Comment:
The issue with checkboxes is not only related to these fields, it's site-wide. We are using [Tailwind Custom Form](https://tailwindcss-custom-forms.netlify.app/) and the `.form-checkbox` class is provided by it. We have some customization in `tailwind.config.js:11` but it looks like it's not enough...